src/Register/Forgot_Id_Pw.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from "react";
+import '../App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Form, Row, Col, Container, Button } from 'react-bootstrap';
+import { Textbox } from 'react-inputs-validation';
+import 'react-inputs-validation/lib/react-inputs-validation.min.css';
```

Comment:
line 5 and 6, these 2 files/module can't be found in this project.

Reply:
fixed it! Thank you for checking!